devops/openshift/forms-deploy.yml

```diff
@@ -122,33 +124,32 @@ objects:
       annotations:
       creationTimestamp: null
       labels:
-        app: ${NAME}
-      name: ${NAME}
+        app: ${HOST_PREFIX}${NAME}
```

Comment:
And this also supports my point here https://github.com/bcgov/SIMS/pull/1574/files#r1037386329
The name must be concatenated almost everywhere. Why it was not concatenated outside and provided as `NAME` here?
Is there anything in the yaml file that will be a problem to have `dev-forms` instead of only `forms`?